src/backend/brokers/go.emails/email.go

```diff
@@ -104,10 +119,9 @@ func (b *EmailBroker) MarshalEmail(msg *Message) (em *EmailMessage, err error) {
 		m.SetHeader("References", strings.Join(ref, " "))
 	}
 
-	m.SetHeader("X-Mailer", "Caliopen-"+b.Config.AppVersion)
-
 	//TODO: In-Reply-To header
 	m.SetHeader("Subject", msg.Subject)
+
 	messages.SanitizeMessageBodies(msg)
 	if msg.Body_html != "" {
 		m.AddAlternative("text/html", msg.Body_html)
```

Comment:
do we still need to add alternative `text/plain` if body is encrypted ? May adding an `if is_encrypted` check

Reply:
if message is encrypted we will not reach this part of code